pages/realms/new.tsx

```diff
@@ -0,0 +1,256 @@
+import Link from 'next/link'
+import { ArrowLeftIcon } from '@heroicons/react/outline'
+import BN from 'bn.js'
+
+import useQueryContext from '@hooks/useQueryContext'
+import Input from '@components/inputs/Input'
+import React, { createContext, useEffect, useState } from 'react'
+import Button from '@components/Button'
+import { RpcContext } from '@models/core/api'
+import { MintMaxVoteWeightSource } from 'models/accounts'
+import { registerRealm } from 'actions/registerRealm'
+import useWalletStore from 'stores/useWalletStore'
+import { PublicKey } from '@solana/web3.js'
+import { notify } from 'utils/notifications'
+import * as yup from 'yup'
+import { formValidation, isFormValid } from '@utils/formValidation'
+import { useRouter } from 'next/router'
+import {
+  ComponentInstructionData,
+  InstructionsContext,
+} from '@utils/uiTypes/proposalCreationTypes'
+import useInstructions from '@hooks/useInstructions'
+
+const publicKeyValidationTest = (value) => {
+  try {
+    if (!value) return false
+    new PublicKey(value)
+    return true
+  } catch (e) {
+    return false
+  }
+}
+const schema = yup.object().shape({
+  governanceProgramId: yup
+    .string()
+    .required('Governance program id is required')
+    .test(
+      'is-public-key',
+      'Governance program id is not a valid public key',
+      publicKeyValidationTest
+    ),
+  name: yup.string().required('Name is required'),
+  communityTokenMint: yup
+    .string()
+    .required('Community token mint is required')
+    .test(
+      'is-public-key',
+      'Community token mint id is not a valid public key',
+      publicKeyValidationTest
+    ),
+})
+
+const defaultGovernanceCtx: InstructionsContext = {
+  instructionsData: [],
+  handleSetInstructions: () => null,
+  governance: null,
+  setGovernance: () => null,
+}
+export const NewProposalContext = createContext<InstructionsContext>(
+  defaultGovernanceCtx
+)
+
+const New = () => {
+  const router = useRouter()
+  const { fmtUrlWithCluster } = useQueryContext()
+  const { getAvailableInstructions } = useInstructions()
+  const availableInstructions = getAvailableInstructions()
+
+  const wallet = useWalletStore((s) => s.current)
+  const connection = useWalletStore((s) => s.connection)
+
+  const [form, setForm] = useState({
+    governanceProgramId: '',
+    name: '',
+    communityTokenMint: '',
+    councilMint: '',
+    communityMintMaxVoteWeightSource: 0.5,
+    minCommunityTokensToCreateGovernance: 0.001,
+  })
```

Comment:
appropriate defaults should be selected here

Reply:
Please use the following defaults:
`communityMintMaxVoteWeightSource`: 1 - by default we want to use full mint supply as max vote weight
`minCommunityTokensToCreateGovernance`: let's use 1% of the community token supply, and if the supply is 0 let's make it arbitrary `1000000` value. 
`governnaceProgramId`: `GovER5Lthms3bLBqWub97yVrMmEogzX7xNjdXpPPCVZw` the default instance

Reply:
ya makes sense - for the 1% of community mint, should we use 1% of council mint if specified?

1% council mint || 1% community mint || 1000000